scripts/test-triton.sh

```diff
@@ -181,39 +186,58 @@ run_unit_tests() {
   lit -v . || $TRITON_TEST_IGNORE_ERRORS
 }
 
+run_pytest_command() {
+  local pytest_command=$1
```

Comment:
Do you really need only the first argument? Or you need the whole command?
Did you mean this?
```suggestion
  local pytest_command="$@"
```

Reply:
> Do you really need only the first argument? Or you need the whole command?

I pass the whole command as one string, so the current version should work as well. Don't you like this approach?

Reply:
Ok, I will try the approach from https://github.com/intel/intel-xpu-backend-for-triton/pull/3891#discussion_r2037865146

Reply:
Ah I see. This should work, now i understand why you build a string with all arguments first (my other comment).